cairocffi/test_xcb.py

```diff
@@ -149,6 +155,11 @@ def test_xcb_pixmap(xcb_conn):
         width, height
     )
 
+    ctx = None
+    surface = None
+    remove_gc(xcb_conn, gc)
+    remove_pixmap(xcb_conn, pixmap)
```

Comment:
I guess when we create the surface, we can monkeypatch the `conn.disconnect()` method to clean up the surfaces before the disconnect happens. That might get a bit messy, tho.